features/aws-ids.yaml

```diff
@@ -1,3 +1,3 @@
-bionic: ami-094e9c95623db463c
+bionic: ami-03b75f613f80bcff1
```

Comment:
I think we should have a separate file for the fips pro images. The rationale for that is because we still want to test the base pro machines

Reply:
Yeah, I was thinking of adding fips machines in the ids. Lemme work on it.